m4_sessions/1808-inning/stress_management.md

```diff
@@ -23,10 +30,24 @@
      Share resources in your slack channel
      All sessions NEED to have a closing activity, reflection, discussion
     
-## Introductons 
+## Introductions
+* who we are
 ## Intro to new material
+* What is stress and how we manage it.
+
 ## Small Group Activity
+after introductions, we will break up into three even groups, each of which will be lead by one of us.
```

Comment:
So students go to all rotations?  I like this idea esp because the smaller group is likely better for most people.